lib/components/term.js

```diff
@@ -195,6 +221,10 @@ export default class Term extends PureComponent {
     // to do in case of splitting, see `componentDidMount`
     this.term._events = {};
 
+    if (process.platform === 'win32') {
+      window.removeEventListener('mousedown', this.mouseHandler);
```

Comment:
If you check for `this.mouseHandler` instead of platform, this code is future proof if someone in the future wanted to make this an option on all platforms 👍 


```js
if (this.mouseHandler) {
  window.removeEventListener('mousedown', this.mouseHandler)
}
```
🙌

Reply:
But wouldn't this always resolve to true?
Even if `this.mouseHandler` is unbound it's still a function of the class. 🤔

Reply:
Oh 🤦‍♂️ you're totally right, there's some more adjustments needed, but that's . probably overkill.

But should this really be default? Should it not be an opt-in setting? 🤔   (im not using windows so i don't know)

Reply:
I went through #2001 and #1247 and it seems like that the `quickEdit` option takes care of that.
Since it's default on windows I thought so. 🙆‍♂️ 
Maybe the `process.platform === 'win32'` should be replace with `this.props.quickEdit`?
Then it should work as it did in before v2.

Reply:
> `on right click selected text will be copied...`

I'll try to add this as well to make it complete. 🚀